### ISSUE TITLE
I can't sort files in the Downloads folder

### ISSUE THREAD

**Issue by dieg467** [tablacus/TablacusExplorer#765]
Hi, I have Windows 10. In the Downloads folder on the C drive, I can't sort files by name, type, size, date modified, etc. When I add a file to this folder, it sorts automatically (it seems to be by date modified or created). Also, the subfolders are not at the top of the list as usual, they appear interspersed in the file list.
This only happens to me with the Downloads folder. In all other folders and subfolders on the C drive I can sort files and subfolders by any criteria I want.
What could be blocking the sorting in the Downloads folder? Thank you very much for the help.

**Comment by ccfs** [comment]
Try clearing [Options > List > Style > File Explorer Options](https://github.com/tablacus/TablacusExplorer/wiki/Options-List-Style-%E2%80%90-File-Explorer-Options).
If this does not work, please copy your TE info from Help > About and paste it in your reply for further investigation.

**Comment by dieg467** [comment]
Thanks for the reply ccfs. I followed the steps indicated and now the sorting of the files is unlocked. However, they are not completely sorted, because the subfolders are still interspersed with the files. If I choose to sort by type, for example, the .pdf files are not all together, but in several groups.

TE64 24.7.28 Microsoft Windows 10 Pro 64 bits 22H2 (10.0.19045) IE/11 JS/11.0.16384 es_es 96 AMD A10-7860K Radeon R7, 12 Compute Cores 4C+8G Windows Defender
mouse 1.33,key 1.26,bgimage 1.09,usercss 1.08,switchpane 1.05,addressbar 1.73,treeview 1.47,systemfolders 1.12,rotatepane 1.03,favorites 1.06,favbar 1.28,favoritesnewtab 1.02,quicknotes 1.02,quicklook 1.11,flat 1.23,toolbar 1.44,mainmenubutton 1.17,controlpanelbutton 1.10,linkbar 1.33,drivebar 1.19,layout 1.02,refresh 1.10,searchbar 1.12,split 1.26,split3 1.12,split6 1.10,emptyrecyclebin 1.06,titlebar 1.09,multithread 1.29,remember 1.42,dragdrop 1.03,inactivepane 1.11,tabplus 1.97,tabcolor 1.09,preventduplicatetabs 1.19,sorttabs 1.04,tabcolorplus 1.07,closealltabs 1.00,tabposition 1.10,font 1.25,hotbutton 1.13,menuimageplus 1.04,statusbar 1.19,tabname 1.09,closewhendeleting 1.03,runatsetup 1.04,openinstead 1.14,multiprocess 1.37,segoeicons 1.04,clipfolder 1.28,multipleexecutions 1.09,label 1.45,colorlabels 1.12,overlaylabel 1.01,startup 1.11,duplicatecopy 1.11,iconlabelplus 1.02

**Comment by ccfs** [comment]
The folder is probably in a "grouping" view. Please choose "Group by > None" after you right-click file list background, like this:
![a](https://github.com/user-attachments/assets/62eb3126-ca24-42fe-9906-2ac2673b8841)

If this still does not work, please post some screen captures showing the problem in your reply for further investigation.

**Comment by dieg467** [comment]
Hi, ccfs. The Downloads folder is not in a Grouping view.
However, I was able to solve the problem by going to Options -> List -> Styles and unchecking, one by one, the options that I highlighted in the red boxes, restarting Tablacus, and then checking them.

![EstilosTablacus](https://github.com/user-attachments/assets/d85aee72-35cd-4169-a250-377c065c4826)

I don't know exactly which of the steps I followed solved the problem. Now the Downloads folder responds to the various sorting options (by name, type, date modified, etc.).
Thank you very much for your help. It guided me to discover the solution.
Greetings!
